dev/README_RELEASE_AIRFLOW.md

```diff
@@ -583,15 +583,14 @@ Check if sources are the same as in the tag:
 ```bash
 cd X.Y.Zrc1
 tar -xvzf *-source.tar.gz
-pushd apache-airflow-X.Y.Zrc1
+pushd apache-airflow-X.Y.Z
 diff -r airflow "${SOURCE_DIR}"
-popd && rm -rf apache-airflow-X.Y.Zrc1
+popd && rm -rf apache-airflow-X.Y.Z
 ```
 
 The output should only miss some files - but they should not show any differences in the files:
 
 ```
```

Comment:
```suggestion
diff -r airflow ~/code/airflow/
```

Reply:
Note @jedcunningham -> no idea how to suggest ``` in the suggestion so just do it manually

Reply:
See https://github.com/apache/airflow/pull/31951#discussion_r1232405802